autoload/slime.vim

```diff
@@ -3,7 +3,8 @@
 """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 
 function! s:TargetSend(config, text)
-  let output = system("CONFIG=" . shellescape(a:config) . " " . g:slime_target_send, a:text)
+  let TargetSend = function(g:slime_target . "#send")
+  let output = TargetSend(a:config, a:text)
```

Comment:
I was imagining `TargetSend` to be either:
- a string: the name of the command to `system` to, or
- a function pointer: users could write their own function (if desired) or delegate to a plugin (probably)

going with a convention, like `slime_myplugin#send`, I don't know if it's less flexible … or would allow you to do the same (if you follow the right naming)? 🤔

Reply:
same applies to line 17 below

Reply:
Could we have a configuration parameter? like, something that tells vim-slime whether it should look for a `slime_myplugin#send` function or try to run a `system` call?